src/Components/FilterGraduates/ByEnglishLevel.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from "react";
+import { Dropdown } from "semantic-ui-react";
+
+class FilterByEnglishLevel extends Component {
```

Comment:
It's necessary FilterByEnglishLevel be a Class?

Reply:
I think that instead of a class component it can be a functional component